src/test/regress/output/multi_copy.source

```diff
@@ -304,6 +304,28 @@ SELECT count(*) FROM pg_dist_shard WHERE logicalrelid = 'lineitem_copy_append'::
      5
 (1 row)
 
+-- Test round robin shard policy
```

Comment:
Actually WorkerGetNodeWithName() uses hash_seq_search(). It was using the second port in regression tests, but the first port on my local tests.